ui/imports/utils/Constants.qml

```diff
@@ -756,6 +756,13 @@ QtObject {
         NoError
     }
 
+    enum AddAccountType {
+        GenerateNew,
+        ImportSeedPhrase,
+        ImportPrivateKey,
+        WatchOnly
```

Comment:
What is this? ;)

Reply:
Moved from SelectGeneratedAccount.qml. It was being used in RootStore AddAccountModal, DerivedAddressPanel. We had a cyclic dependency on this because it was being used in the wallet singleton RootStore:
`Warning: Cyclic dependency detected between "qrc:/app/AppLayouts/Wallet/stores/RootStore.qml" and "qrc:/app/AppLayouts/Wallet/panels/SelectGeneratedAccount.qml" (:0, )`

Reply:
I see, cool